.github/workflows/bump_release_version.yml

```diff
@@ -0,0 +1,30 @@
+#############################################################################
+# GitHub Action to bump release version
+#
+#############################################################################
+name: "Bump version"
+on:
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  bump:
+    name: Bump version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Modify File
+        run: |
+          RELEASE_VERSION=$(git describe --abbrev=0 | cut -d '-' -f 2)
```

Comment:
please use this approach - https://github.com/orgs/community/discussions/26686#discussioncomment-3252853

Reply:
I am not getting the desired output with this tag. I was expecting `10.3.1` but this command is giving me the latest release which is 10.4-SNAPSHOT

Reply:
Look at https://github.com/checkstyle/checkstyle/pull/11993#discussion_r932556123 it might be a reason.

Reply:
this action can use human trigger, se comment below